packages/text-bitmap/src/BitmapText.ts

```diff
@@ -65,6 +70,7 @@ const charRenderDataPool: CharRenderData[] = [];
  */
 export class BitmapText extends Container
 {
+    private static msdfProgram = new Program(msdfVert, msdfFrag);
```

Comment:
no need to store this as we can leverage the Program Cache `Program.from(msdfVert, msdfFrag)`